samples/languages/nodejs/nodejs-enclave_config.json

```diff
@@ -0,0 +1,14 @@
+{
+    "args": [
+      "/usr/bin/node"
+      "app/cpubench.js"
```

Comment:
I can't see cpubench.js as part of this PR. Is it generated automaically?

Reply:
Oops, that was missing. Now added.